notebooks/Media-Pretrained/02_Training_Layer_Recap.ipynb

```diff
@@ -17,49 +17,25 @@
-    "\n",
-    "Insert IMDB Data descriptoin\n",
-    "\n",
-    "\n",
-    "[User Metadata](https://docs.aws.amazon.com/personalize/latest/dg/users-datasets.html) The user data that you can import into Amazon Personalize includes numerical and categorical metadata about your users, such as gender or loyalty membership. Since we do not have user data for the users in the Movielens data set, we are generating a synthetic dataset for the purposes of this workshop.\n",
+    "## Introduction <a class=\"anchor\" id=\"intro\"></a>\n",
     "\n",
-    "1. **UserID** - The user who interacted\n",
-    "1. **Membershiplevel** - A randomly assigned membership level\n",
+    "In the previous notebook we prepared 3 different datasets that represent sample data that would exist in a Media & Entertainment applicatiopn (User interactions, Media catalog data and subscriber/user data). In order to complete this workshoop within the time set, we have already created several resources on your behalf. \n",
     "\n",
     "### In this notebook we will accomplish the following:\n",
     "\n",
-    "Create Video on Demand Domain Recommenders for the following use cases:\n",
+    "Walk you throuw how we created Video on Demand Domain Recommenders for the following use cases:\n",
     "\n",
     "1. [More like X](https://docs.aws.amazon.com/personalize/latest/dg/VIDEO_ON_DEMAND-use-cases.html#more-like-y-use-case): recommendations for movies that are similar to a movie that you specify. With this use case, Amazon Personalize automatically filters movies the user watched based on the userId that you specify and Watch events.\n",
     "\n",
     "1. [Top picks for you](https://docs.aws.amazon.com/personalize/latest/dg/VIDEO_ON_DEMAND-use-cases.html#top-picks-use-case): personalized content recommendations for a user that you specify. With this use case, Amazon Personalize automatically filters videos the user watched based on the userId that you specify and Watch events.\n",
     "\n",
-    "Create a custom solution and solution versions for the following use case:\n",
+    "Walk you throuw how we created custom solution and solution versions for the following use case:\n",
     "\n",
     "3. [Personalized-Ranking](https://docs.aws.amazon.com/personalize/latest/dg/working-with-predefined-recipes.html): will be used to rerank a list of movies.\n",
     "\n",
-    "![Workflow](images/image2.png)\n",
-    "\n",
-    "To run this notebook, you need to have run the previous notebook, `01_Data_Layer.ipynb`, where you created a dataset and imported interaction, item, and user metadata data into Amazon Personalize. At the end of that notebook, you saved some of the variable values, which you now need to load into this notebook."
+    "![Workflow](images/image2.png)\n"
```

Comment:
It would be helpful to introduce or caption this diagram.  Something like:

The following diagram shows the resources that we will create in this section, highlighted in blue.

Reply:
Fixing!